docs/contributors/e2e_tests.md

```diff
@@ -0,0 +1,57 @@
+# Execute end-to-end tests for local development
```

Comment:
It's super helpful to have this documented, but its even better if the e2e suite setup could do these steps automatically. Is that something you considered?

For example: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/test/e2e/ctx/provisioner_kind.go

Reply:
When I update the code in my next push this will be resolved since I have implemented `ctx` similar to what is done in OLM.